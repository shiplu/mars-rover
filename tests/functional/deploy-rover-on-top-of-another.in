5 3
2 2 E
MMMRM
2 2 N
MMMLM
