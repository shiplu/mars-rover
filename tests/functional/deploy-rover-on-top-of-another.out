5 1 S OPERATIONAL
